### DISCUSSION TITLE
О библиотеке ⚡️ Angular с примерами кода

### DISCUSSION THREAD

**Discussion by giscus** [xsltdev/angdev.ru#58] [Announcements]
# О библиотеке ⚡️ Angular с примерами кода

RxJS представляет собой библиотеку, позволяющую управлять всеми асинхронными операциями и событиями в приложении в стиле реактивного программирования

https://angdev.ru/rxjs/about/

**Comment by nikitasmirnovveis** [comment]
картинок бы
